en/docs/samples/configuring-sso-oidc-sample.md

```diff
@@ -0,0 +1,41 @@
+# Configuring Single Sign On Using OpenID Connect
+
+This page guides you through configuring single sign on authentication between two OAuth/OpenID Connect web applications using two **sample applications** called Pickup Dispatch and Pickup Manager. 
+
+----
+
+## Scenario
+
+Pickup is a cab company that has two SAML web applications called pickup-dispatch and pickup-manager. Both applications use WSO2 IS as the identity provider. When SSO is configured for both these applications, a user is only required to provide their credentials to the first application and the user will be automatically logged in to the second application.
+
+![oidc-sso-scenario](../assets/img/samples/oidc-sso-scenario-diagram.png)
+
+Follow the steps below to deploy two sample applications and see how this works. 
+
+----
+
+## Set up Pickup Dispatch sample
+
+{!samples/pickup-dispatch-oidc.md!}
+
+----
+
+## Set up Pickup Manager sample
+
+{!samples/pickup-manager-oidc.md!}
+
+You are now ready to try out OpenID Connect SSO with the Pickup Dispatch and Pickup Manager sample web applications.
+
+----
+## Try it out
+
+1. Navigate to <http://localhost.com:8080/pickup-dispatch> on your browser and click **Login**.
```

Comment:
Didn't we decide to use our identity server-specific domain name instead of `localhost.com`?

Reply:
Yes. Fixed with af159276859f62cacdd93dabd91d00245ee70655